mqtt-client-Elixir/mqtt_tls.exs

```diff
@@ -0,0 +1,52 @@
+Mix.install([
+  {:emqtt, github: "emqx/emqtt", tag: "1.8.2"},
+  {:castore, "~> 1.0.0"}
+])
+
+require Logger
+
+Process.flag(:trap_exit, true)
+
+## Connect via SSL and verify the server certificate
+
+{:ok, subscriber} = :emqtt.start_link(
+  ## Note single quotes
+  host: 'broker.emqx.io',
+  port: 8883,
+  client_id: "elixir_subscriber",
+  username: "user",
+  password: "password",
+  ssl: true,
+  ssl_opts: [
+    verify: :verify_peer,
+    server_name_indication: 'broker.emqx.io',
+    cacertfile: CAStore.file_path()
```

Comment:
Q: is this just an example?  Or were this module supposed to exist?  I couldn't find it.  :see_no_evil:

Reply:
This module belongs to `castore` hex, insalled in the first line

Reply:
Oops, glossed over that line, sorry :see_no_evil: